internal/store/model/source.go

```diff
@@ -21,6 +21,7 @@ type Source struct {
 	SshKey     string
```

Comment:
We we want to move the sshkey out of here. Or will that be done in followup PRs?

Reply:
follow up